torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5759,6 +5759,28 @@ def sample_inputs_atleast1d2d3d(op_info, device, dtype, requires_grad, **kwargs)
     samples.append(SampleInput(input_list, ))
     return samples
 
+def sample_inputs_column_stack(op_info, device, dtype, requires_grad, **kwargs):
+    input_list = []
+    cases: Tuple[tuple, tuple] = (  # type: ignore[assignment]
+        ((S, 2, 1), (S, 3, 1)),
```

Comment:
nit: one line per test case

Reply:
Each test case is a list of tensors, so we already have one line per test case

Reply:
My mistake